meta-lmp-bsp/dynamic-layers/xilinx-standalone/recipes-bsp/embeddedsw/pmu-firmware_2022.1.bbappend

```diff
@@ -1,2 +1,5 @@
 # Allow EFUSE access, used by OP-TEE
 YAML_COMPILER_FLAGS:zynqmp = " -DENABLE_EFUSE_ACCESS"
```

Comment:
Probably is better to have an append there to don't overwrite the default upstream
`YAML_COMPILER_FLAGS:append:zynqmp = " -DENABLE_EFUSE_ACCESS"
`

Reply:
There might be either:
```
YAML_COMPILER_FLAGS:zynqmp += " -DENABLE_EFUSE_ACCESS"
```
or
```
YAML_COMPILER_FLAGS:append:zynqmp = "-DENABLE_EFUSE_ACCESS"
```

Reply:
@quaresmajose As Yocto RM recommends, it's better to use `append` as a safe way to avoid conflicts with later hard assigns `=` and weak assigns `?=`.

Reply:
Yup, good catch, will fix.